src/render/BpmnPropertiesPanel.js

```diff
@@ -207,7 +221,7 @@ export default function BpmnPropertiesPanel(props) {
       headerProvider={ PanelHeaderProvider }
       placeholderProvider={ PanelPlaceholderProvider(translate) }
       groups={ groups }
-      layoutConfig={ layoutConfig }
+      layoutConfig={ layout }
```

Comment:
Would make sense to change this key to `layout`, too? 

Otherwise let's stay consistent and keep `layout` named `layoutConfig`.